components/fxa-client/src/internal/device.rs

```diff
@@ -295,11 +293,7 @@ impl FirefoxAccount {
     // for the device because the server does not have a `PATCH commands`
     // endpoint yet.
     #[allow(dead_code)]
```

Comment:
I suspect these `dead_code` annotations are now stale, so we should probably get an issue on file to remove them, as IMO they have value in general. I understand they are often necessary during dev, but generally shouldn't be necessary longer term.

Reply:
I've removed a couple, and put a couple back that appeared to still be necessary.